falco.yaml

```diff
@@ -444,66 +444,86 @@ metadata_download:
   watch_freq_sec: 1
 
 
-# base_syscalls ! Use with caution !
+# base_syscalls ! Use with caution, read carefully !
```

Comment:
Tried to make the description easier to follow, we can plan for some final touch up closer to Falco 0.35 release.